services/movie_session.py

```diff
@@ -0,0 +1,61 @@
+from typing import Optional
+
+from db.models import MovieSession
+from django.db.models import QuerySet
+
+import datetime
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    new_movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+    return new_movie_session
+
+
+def get_movies_sessions(
+        movie_show_time: Optional[str] = None
+) -> QuerySet:
+    query_set = MovieSession.objects.filter()
+    if movie_show_time:
+        query_set = query_set.filter(
+            show_time__date=movie_show_time
+        )
+
+    return query_set
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: Optional[datetime] = None,
```

Comment:
datetime is module
Use datetime.datetime here

Reply:
I did it in last commit